### DISCUSSION TITLE
今晚整了个另一个项目的机器人，还挺好看的

### DISCUSSION THREAD

**Discussion by Xm798** [Arondight/Adachi-BOT#763] [General]
项目是 Yunzai-Bot
我竟然在同一个QQ上同时挂了三个项目，Adachi-Bot Yunzai-Bot和HarukaBot，分别用了iPad Mac和手表协议，备用手机上还挂了个安卓，真是榨干了这个QQ号的所有资源……
另外阿里云的1c2g小机器没能顶住……分了一个到腾讯云的4c4g机器上，（140买了三年，真便宜）

![image](https://user-images.githubusercontent.com/48074861/160654454-3ac70c0a-4454-40d6-a182-b1fdf9adac65.png)![image](https://user-images.githubusercontent.com/48074861/160654466-d54c7ee2-19c2-4826-93ff-68db1dd801b3.jpeg)

**Comment by mark9804** [comment]
我之前看到过， #621 

他们项目应该也迭代过几版设计

~~所以这个 bot 真的可以做到 100-300ms 的响应时间吗~~

**Reply by Arondight** [reply]
性能指标是个很虚的东西

**Comment by Arondight** [comment]
轻应用？我后悔买少了，68一年

**Reply by Xm798** [reply]
轻量应用服务器，买的时候是2c4g8m，前段时间免费升级4c4g

**Comment by Arondight** [comment]
我看了几个项目，原批bot这块也太卷了，功能五花八门的，而且都很好看，幸好这个项目的第一目的是自用，不用跟他们一起卷了……

**Reply by mark9804** [reply]
挺好的，二创社区活跃，卷就让他们卷吧，我们几个人也卷不过人家
<img src="https://user-images.githubusercontent.com/9006264/160740223-f5f45c11-5fea-4111-9272-0e518a55b480.png" width="200" />

**Comment by DongGua-Jun** [comment]
同QQ挂多个bot特别容易吃风控

**Reply by Xm798** [reply]
问题不大，群不多

**Comment by mark9804** [comment]
今天看到这张图，心动了
![B537BEAD-0C9F-4162-A2BB-36E0C136738B](https://user-images.githubusercontent.com/9006264/170901404-107acb84-3052-49fc-9691-fa65dc081753.jpeg)

**Reply by Xm798** [reply]
查面板确实不错，至今不知道怎么做到的

**Reply by mark9804** [reply]
https://github.com/yoimiya-kokomi/miao-plugin

应该就是这个插件，我已经给 deadline 掏空了，无力分析

**Reply by Xm798** [reply]
> https://github.com/yoimiya-kokomi/miao-plugin
> 
> 应该就是这个插件，我已经给 deadline 掏空了，无力分析

接口不开源，Token现在不发了（还好我拿的早

**Reply by mark9804** [reply]
> 接口不开源

返回数据的速度如何，不知道是不是爬了提瓦特小助手的接口

**Reply by Xm798** [reply]
> > 接口不开源
> 
> 返回数据的速度如何，不知道是不是爬了提瓦特小助手的接口

更新面板如果没有遇到并发限制，八个角色五六秒钟吧

**Reply by mark9804** [reply]
> 八个角色五六秒钟吧

那就好玩起来了，想不到怎么实现的

**Reply by mark9804** [reply]
> 想不到怎么实现的

还是很诚实的去翻了翻源码，跑了个MTR，目标服务器是腾讯云上面的，看 kokomi 的 commit 风格感觉也是学生，不太可能整虚拟机那套资本主义生产方式，分析不下去，只能遗憾一句来晚了

**Reply by DongGua-Jun** [reply]
酸了
